python/cugraph/dask/pagerank/pagerank.py

```diff
@@ -162,7 +162,6 @@ def pagerank(edge_list, alpha=0.85, max_iter=30):
     >>>                                    dtype=['int32', 'int32'])
     >>> pr = dcg.pagerank(ddf_edge_list, alpha=0.85, max_iter=50)
     """
-
```

Comment:
Is there some dead code from SNMG or duplicate from RAFT in this file?

Reply:
This is entirely snmg code. It simply connects to cugraph/python/cugraph/snmg/link_analysis/mg_pagerank.py which throws an exception saying it is disabled. We could update it to throw exception at python/cugraph/dask/pagerank/pagerank.py or just remove it completely for 0.15 release once we have OPG pagerank.